### PR TITLE
Higgs 13.6 tev signal samples

### PR DESCRIPTION
**Pull request by chuwang1**
Dear expert,
We have produced some Higgs analysis madgraph cards at 13.6TeV for RunIII , which included:

ggF cards
VBF cards
ttH cards
VH cards
For these cards, we only changed the beam energy in the run_card.
Please review them, if there are any issues, please tell us.
Thanks in advance!
Chu Wang

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/vbfh_5f_NLO_/vbfh_5f_NLO__FKS_params.dat [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+!
+! This parameter sets the minimal fraction of the events for which the
+! virtual matrix elements should be included.
+#MinVirtualFraction
+0.005d0
+! Default :: 0.005d0
+!
+! ==========================================================================
+! End of FKS_params.dat file
+! ==========================================================================
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/vbfh_5f_NLO_/vbfh_5f_NLO__FKS_params.dat:75]: regarding this FKS file, have you tried the production without it. I checked the previous card was merged 5 years ago, maybe now it's fine to run without it

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/vh012j_5f_NLO_FXFX_M_VToAll/vh012j_5f_NLO_FXFX_M_VToAll_madspin_card.dat [hunk 2]
```diff
@@ -0,0 +1,13 @@
+set ms_dir ./madspingrid
+
+set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/vh012j_5f_NLO_FXFX_M_VToAll/vh012j_5f_NLO_FXFX_M_VToAll_madspin_card.dat:3]: please use Nevents_for_max_weight instead

Comment (chuwang1) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/vh012j_5f_NLO_FXFX_M_VToAll/vh012j_5f_NLO_FXFX_M_VToAll_madspin_card.dat:3]: ok, I will try it. Thanks for pointing out.

**Comment by XuanhaoZhang** [comment]
Hi, Meng. Thanks for the suggestion, we have regenerated grid pack and tested it. If there is any other problem with cards, let us know.

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/vh012j_5f_NLO_FXFX_M_VToAll/vh012j_5f_NLO_FXFX_M_VToAll_madspin_card.dat (hunk 2 above)
Comment (chuwang1) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/vh012j_5f_NLO_FXFX_M_VToAll/vh012j_5f_NLO_FXFX_M_VToAll_madspin_card.dat]: Dear Meng,
We have updated the cards, please review them again.

File: bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/vbfh_5f_NLO_/vbfh_5f_NLO__FKS_params.dat (hunk 1 above)
Comment (DickyChant) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/vbfh_5f_NLO_/vbfh_5f_NLO__FKS_params.dat]: Hi just a reminder: FKS is needed for this NLO VBF processes

Comment (DickyChant) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/vbfh_5f_NLO_/vbfh_5f_NLO__FKS_params.dat]: For NLO VBF we need to bypass the pole checking

Comment (chuwang1) [reply bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/Higgs/vbfh_5f_NLO_/vbfh_5f_NLO__FKS_params.dat]: Thanks! @XuanhaoZhang and I will add this file after local validation.

**Pull request merged**